core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly

```diff
@@ -32,10 +32,10 @@ THE SOFTWARE.
     <j:getStatic var="createPermission" className="hudson.model.Computer" field="CREATE"/>
     <l:tasks>
       <l:task href="${rootURL}/" icon="icon-up icon-md" title="${%Back to Dashboard}"/>
```

Comment:
We should probably generalize common sidepanel controls to avoid different behavior on screens. It will likely also help @fqueiruga with the sidepanel restructuring proposal. Not required for this PR

Reply:
I agree. I have no idea how to go about it though